.gitignore

```diff
@@ -0,0 +1,5 @@
+.*.swo
+.*.swp
+
+server_standalone/server_standalone
```

Comment:
Yes, I did my integration tests that way. I probably could have used go install but it's just habit...